WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -649,13 +650,13 @@ public static boolean shouldShowGutenbergInfoPopupForTheNewPosts(String siteURL)
         return flag;
     }
 
-    public static void setShowGutenbergInfoPopupForTheNewPosts(String siteURL, boolean show) {
+    public static void setBooleanPrefForKeyAndURL(PrefKey key, String siteURL, boolean show) {
```

Comment:
Kinda nit but, I think that the method name has gotten a bit too generalized, to the point that is probably not making too much sense. Can we keep the previous name or is there some reason to move away from that one?

Reply:
Updated in 4f4cdb2278e547207c78f2105bf0c4ce0d9b3e10